src/enchcracker/EnchCrackerWindow.java

```diff
@@ -701,6 +726,14 @@ public void mouseExited(MouseEvent e) {}
 						List<Enchantments.EnchantmentInstance> enchantments = Enchantments
 								.getEnchantmentsInTable(rand, xpSeed, itemToEnch[0], slot, enchantLevels[slot], mcVersion);
 
+						if (enchantLevels[slot] == 0) {
+							continue slotLoop;
+						} else if (i == -1 && playerLevel < enchantLevels[slot]) {
+							continue slotLoop;
+						} else if (playerLevel < (enchantLevels[slot] + 1)) {
+							continue slotLoop;
```

Comment:
These if statements don't correctly handle the case where `i == -1 && playerLevel == enchantLevels[slot]`